### PR TITLE
chore(crashtracker): add profiler_config tag if profiler enabled

### PR DESCRIPTION
**Pull request by taegyunkim**
The previous attempt set the tag after crashtracker has started, and tags didn't show up in the logs. 

## Checklist
- [x] PR author has checked that all the criteria below are met
- The PR description includes an overview of the change
- The PR description articulates the motivation for the change
- The change includes tests OR the PR description describes a testing strategy
- The PR description notes risks associated with the change, if any
- Newly-added code is easy to change
- The change follows the [library release note guidelines](https://ddtrace.readthedocs.io/en/stable/releasenotes.html)
- The change includes or references documentation updates if necessary
- Backport labels are set (if [applicable](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting))

## Reviewer Checklist
- [x] Reviewer has checked that all the criteria below are met 
- Title is accurate
- All changes are related to the pull request's stated goal
- Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes
- Testing strategy adequately addresses listed risks
- Newly-added code is easy to change
- Release note makes sense to a user of the library
- If necessary, author has acknowledged and discussed the performance implications of this PR as reported in the benchmarks PR comment
- Backport labels are set in a manner that is consistent with the [release branch maintenance policy](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting)

### CODE REVIEW

**Review by emmettbutler — APPROVED** [review]

File: ddtrace/internal/core/crashtracking.py [hunk 1]
```diff
@@ -57,6 +59,9 @@ def start() -> bool:
     for key, value in crashtracker_config.tags.items():
         add_tag(key, value)
 
+    if profiling_config.enabled:
+        add_tag("profiler_config", config_str(profiling_config))
```
Comment (brettlangdon) [inline ddtrace/internal/core/crashtracking.py:63]: can profiling be enabled/disabled at runtime via remote config?

Comment (taegyunkim) [reply ddtrace/internal/core/crashtracking.py:63]: I am not aware of such functionality. @sanchda do we have this?

**Review by sanchda — COMMENTED** [review]
Looks fine to me.  This does change extension loading a bit, but I think the way it does so is pretty tame (from an overhead perspective).

File: ddtrace/internal/core/crashtracking.py (hunk 1 above)
Comment (brettlangdon) [reply ddtrace/internal/core/crashtracking.py:63]: not a problem, was mostly trying to explore if we would end up in a state where we don't set this config but profiling is enabled.

as long as it is startup only decision, then I think we are good here?

**Review by danielsn — APPROVED** [review]

**Pull request merged**